packages/veraswap-app/src/atoms/stargate.ts

```diff
@@ -81,16 +86,16 @@ export const stargateQuoteAtom = atomWithQuery((get) => {
         };
 
         return stargateETHQuoteQueryOptions(config, params);
-    }
+    } else {
```

Comment:
for added safety, maybe replace `bridgeSymbol === "ETH"` with `bridge.currencyIn.isNative`